src/hci.cpp

```diff
@@ -1714,7 +1714,6 @@ INT_RETVAL intRunWidgets()
 
 						if (psBuilding->type == REF_DEMOLISH)
 						{
-							STRUCTURE tmp(0, selectedPlayer);
```

Comment:
This value is not used anywhere. Did I understand correctly that I can delete it?

Reply:
Yes, it is correct.